controller/array_action/messages.py

```diff
@@ -1,70 +1,71 @@
-NoConnectionAvailableException_message = "Currently no connection is available to endpoint: {0}"
+NO_CONNECTION_AVAILABLE_EXCEPTION_MESSAGE = "Currently no connection is available to endpoint: {0}"
```

Comment:
Do we don't want to remove the upper case?

Reply:
it's a const and this is the common convention for consts

Reply:
I meant for the msg

Reply:
oh, sure, but I think there is enough diff in this PR.
we can align all messages content in a single separate PR

Reply:
can you open a ticket?

Reply:
CSI-3899